### PR TITLE
Add styletron example

### PR DESCRIPTION
**Pull request by nkzawa**

### CODE REVIEW

File: examples/with-styletron/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
+# Styletron example
+
+## How to use
+
+Download the example:
```
Comment (impronunciable) [inline examples/with-styletron/README.md]: We changed the structure of the readmes https://github.com/zeit/next.js/pull/485/files#diff-481c8b6dd98d2038ce05b93afea6b60d

File: examples/with-styletron/next.config.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  webpack: function (config) {
```
Comment (ads1018) [inline examples/with-styletron/next.config.js]: Out of curiosity, what does this webpack config override do?

Comment (sergiodxa) [inline examples/with-styletron/next.config.js]: It's setting `styletron-server` as an external dependency, that way webpack will not add it to the bundle and instead it's going to get it from `node_modules`

Comment (ads1018) [inline examples/with-styletron/next.config.js]: Gotcha. Thanks.

**Pull request merged**